package.json

```
@@ -28,7 +28,8 @@
     "preversion": "npm test",
     "version": "npm run build-prod && git add release/",
     "postversion": "git push origin master",
-    "deploy": "git push 5apps master"
+    "deploy": "git push 5apps master",
+    "sockethub": "DEBUG=sockethub* ./node_modules/sockethub/bin/sockethub"
```

As sockethub defines an executable to be installed, and executables from dependencies can be used in npm scripts, this should just read:

```suggestion
    "sockethub": "DEBUG=sockethub* sockethub"
```